### PR TITLE
Fixes 1432: Fix prototype exploration answer

### LINKED ISSUE #1432: Incorrect answers in Prototype exploration
In Prototype exploration, the 7th and 8th questions have a theoretically wrong answer.
Right now correct answer is 
`3/5 0.5 0.35 0.46`
whereas it should be 
`3/5 0.5 0.46 0.35`

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
## Explanation

Fixes #1432 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt [hunk 1]
```diff
@@ -705,7 +705,7 @@ class StateFragmentTest {
     onView(withId(R.id.submit_answer_button)).perform(click())
     onView(withId(R.id.continue_navigation_button)).perform(click())
 
-    // Seventh state: Drag Drop Sort. Correct answer: Move 1st item to 3rd position.
+    // Seventh state: Drag Drop Sort. Correct answer: Move 1st item to 4rd position.
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt]: 4th

File: domain/src/main/assets/prototype_exploration.json [hunk 2]
```diff
@@ -849,7 +849,7 @@
       },
       "content": {
         "content_id": "content",
-        "html": "<p>Sort the following in ascending order, putting equal items in the same position.</p>"
+        "html": "<p>Sort the following in descending order, putting equal items in the same position.</p>"
```
Comment (BenHenning) [inline domain/src/main/assets/prototype_exploration.json:852]: I think that the first drag & drop state also says 'putting equal items in the same position' but grouping isn't enabled for it. Can you please fix the text accordingly?

File: app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt (hunk 1 above)
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt]: You added another action here (e.g. you seem to be moving two things here); please clarify in the comment.

**Review by BenHenning — APPROVED** [review]
Thanks for the quick fix @aggarwalpulkit596! LGTM minus some nits.

File: domain/src/main/assets/prototype_exploration.json (hunk 2 above)
Comment (aggarwalpulkit596) [reply domain/src/main/assets/prototype_exploration.json:852]: done

File: app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt (hunk 1 above)
Comment (aggarwalpulkit596) [reply app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt]: done

**Pull request merged**